xrootd/xrootd_config_test.go

```diff
@@ -61,10 +78,63 @@ func TestXrootDCacheConfig(t *testing.T) {
 
 	dirname := t.TempDir()
 	viper.Reset()
+	config.InitConfig()
 	viper.Set("Xrootd.RunLocation", dirname)
 	configPath, err := ConfigXrootd(ctx, false)
 	require.NoError(t, err)
 	assert.NotNil(t, configPath)
+
+	t.Run("TestCacheThrottlePluginEnabled", func(t *testing.T) {
+		xrootd := xrootdTest{T: t}
+		xrootd.setup()
+
+		// Set our config
+		viper.Set("Cache.Concurrency", 10)
+		dirname := t.TempDir()
+		viper.Set("Xrootd.RunLocation", dirname)
+
+		// Generate the xrootd config
+		configPath, err := ConfigXrootd(ctx, false)
+		require.NoError(t, err)
+		assert.NotNil(t, configPath)
+
+		// Verify the output
+		file, err := os.Open(configPath)
+		assert.NoError(t, err)
+		defer file.Close()
+
+		content, err := io.ReadAll(file)
+		assert.NoError(t, err)
+		assert.Contains(t, string(content), "xrootd.fslib throttle default")
+		assert.Contains(t, string(content), "throttle.throttle concurrency 10")
+		viper.Reset()
+	})
+
+	t.Run("TestCacheThrottlePluginDisabled", func(t *testing.T) {
+		xrootd := xrootdTest{T: t}
+		xrootd.setup()
+
+		// Set our config
+		dirname := t.TempDir()
+		viper.Set("Xrootd.RunLocation", dirname)
+
+		// Generate the xrootd config
+		configPath, err := ConfigXrootd(ctx, false)
+		require.NoError(t, err)
+		assert.NotNil(t, configPath)
+
+		// Verify the output
+		file, err := os.Open(configPath)
+		assert.NoError(t, err)
+		defer file.Close()
+
+		content, err := io.ReadAll(file)
+		assert.NoError(t, err)
+		assert.NotContains(t, string(content), "xrootd.fslib throttle default")
+		assert.NotContains(t, string(content), "throttle.throttle concurrency 10")
```

Comment:
```suggestion
		assert.NotContains(t, string(content), "throttle.throttle concurrency")
```
I suspect if the text is wrongly in there, then the number is not likely to be 10.